.github/workflows/playwright.yml

```diff
@@ -83,8 +115,13 @@ jobs:
         shell: bash
 
       - name: Install Playwright Browsers
+        if: ${{ !inputs.dockerized-playwright-prepare }}
         run: npx playwright install --with-deps chromium
 
+      - name: Prepare Playwright environment in container
+        if: ${{ inputs.dockerized-playwright-prepare }}
+        run: ${{ inputs.dockerized-playwright-prepare }}
```

Comment:
I don't think we should allow running inputs directly, this could lead to security issues. What's an example of the commad that this `dockerized-playwright-prepare` input should contain?

Reply:
It's a prepare command just like on bare metal we need to run the `Install Playwright Browsers` before running playwright, so in this case this would do a similar thing but for the dockerized one for example we would pass an npm script like this:

```
"e2e:ci:prepare": "docker build --progress=auto -t metrics-drilldown-plugin-e2e:latest -f e2e/docker/Dockerfile.plugin .",
```

Reply:
I do understand the security issues, we're also okay with removing the dockerized playwright and just keeping the other enhancements for now